source/boundary_traction/ascii_data.cc

```diff
@@ -35,10 +35,12 @@ namespace aspect
     void
     AsciiData<dim>::initialize ()
     {
-      for (const auto &bv : this->get_boundary_traction())
-        if (bv.second.get() == this)
-          boundary_ids.insert(bv.first);
-
+      for (const auto &bv : this->get_boundary_traction_manager ().get_active_boundary_traction_conditions ())
```

Comment:
remove the spaces between function name and parentheses:
```suggestion
      for (const auto &bv : this->get_boundary_traction_manager().get_active_boundary_traction_conditions())
```